aspnetcore-runtime-60/snap/hooks/post-refresh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh -e
+
+if [ -e "$SNAP_COMMON"/dotnet-installer ]; then
+    touch "$SNAP_COMMON"/dotnet-installer-update-mounts
```

Comment:
**question:** Regarding _"The dotnet-installer-update-mounts file will trigger the update mechanism on the dotnet-installer tool [...]"_: Where is the implementation for that? I could not find the implementation if it already exists (see the search I ran: https://github.com/search?q=org%3Acanonical%20dotnet-installer-update-mounts&type=code). Is this planned to be implemented?

Reply:
It's implemented in the .NET installer. [This](https://github.com/canonical/dotnet-snap/blob/feat/new-structure/src/Dotnet.Installer.Console/Scripts/%7BSNAP%7D-update-watcher.path) systemd-path unit will watch for the existence of `dotnet-installer-update-mounts` and trigger [this](https://github.com/canonical/dotnet-snap/blob/feat/new-structure/src/Dotnet.Installer.Console/Scripts/%7BSNAP%7D-update-watcher.service) service, which runs [this](https://github.com/canonical/dotnet-snap/blob/feat/new-structure/src/Dotnet.Installer.Console/Scripts/update-mounts.sh) script to update the mounts.

Reply:
Thanks! Weird that the GitHub search didn't showed that.